addons/packages/vsphere-csi/2.4.0/bundle/config/values.yaml

```diff
@@ -19,4 +19,5 @@ vsphereCSI:
   http_proxy: ""
   https_proxy: ""
   no_proxy: ""
+  cluster_plan: prod
```

Comment:
Probably just call this deployment_replicas and set default to 3. In tanzu-framework adjust the replicas based on plan.

Reply:
ACK

Reply:
Changed.